backend/src/main/java/de/lowani/backend/api/User.java

```diff
@@ -0,0 +1,25 @@
+package de.lowani.backend.api;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class User {
+
+    @ApiModelProperty(required = true, example = "234", notes = "The ID of the user")
```

Comment:
Do you need the ID in the User? It's generated automatically in UserEntity and probably not used outside your database?

Reply:
The UserID will be needed for the question History, since it shouldn't show any names. Maybe I find another way